### ISSUE TITLE
Idea: Map mobility data from construction areas to hospital regions

### ISSUE THREAD

**Issue by atsf42** [datapartnership/heatwaves#12]
Q: would someone cal the hospital from the construction area if they are suffering from a heat stroke? What about neighboring calls?

**Comment by g4brielvs** [comment]
@atsf42 That is an interesting idea. If we gain access to geolocation data, it may be possible to visualize overall movement (not only to hospitals) within a city during a heatwave (given limitations).

**Comment by g4brielvs** [comment]
@atsf42 @SahitiSarva [Measuring impact of excess heat in urban areas](https://portal.datapartnership.org/submitproposal/466) has been updated with Veraset. @jfulponi Could you please work getting the approval on https://github.com/datapartnership/operations/issues/617?

**Comment by jfulponi** [comment]
That's right! Measuring overall movement within a city during a heatweave is pretty easy if we have the right temporal range. I think we can try making some polygon buffers according to hospital regions to have a more specific indicator too.
